### PR TITLE
fix: make bitmap to vairiant more understandable

### LINKED ISSUE #13051: bug: cast bitmap to variant return nonsense characters
### Search before asking

- [X] I had searched in the [issues](https://github.com/datafuselabs/databend/issues) and found no similar issues.

### Version

v1.2.132-nightly

### What's Wrong?

cast bitmap to variant return nonsense characters
we should first convert the bitmap  value to an array of int, and then convert to jsonb value

### How to Reproduce?

```sql
mysql> create table test(a bitmap);
Query OK, 0 rows affected (0.07 sec)

mysql> insert into test values('100,200'),('50,60,90');
Query OK, 2 rows affected (0.52 sec)

mysql> select to_string(a), to_variant(a) from test;
+--------------+------------------------------------+
| to_string(a) | to_variant(a)                      |
+--------------+------------------------------------+
| 100,200      |            :0           d �    |
| 50,60,90     |            :0           2 < Z  |
+--------------+------------------------------------+
2 rows in set (0.03 sec)
Read 2 rows, 91.00 B in 0.009 sec., 229.87 rows/sec., 10.21 KiB/sec.
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by ct20000901**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Making the bitmap conversion to a variant more understandable.
image

- Closes #13051

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/13077)

### CODE REVIEW

File: tests/sqllogictests/suites/query/02_function/02_0048_function_semi_structureds_parse_json [hunk 1]
```diff
@@ -222,3 +222,28 @@ select to_variant(a), to_variant(b), to_variant(c) from t3
...
+CREATE DATABASE db2
+
+statement ok
+USE db2
+
+statement ok
+CREATE TABLE IF NOT EXISTS test(a bitmap) Engine = Fuse
+
+statement ok
+insert into test values('100,200'),('50,60,90');
```
Comment (b41sh) [inline tests/sqllogictests/suites/query/02_function/02_0048_function_semi_structureds_parse_json:239]: move those tests to `tests/sqllogictests/suites/query/02_function/02_0002_function_cast` is better.

Comment (ct20000901) [reply tests/sqllogictests/suites/query/02_function/02_0048_function_semi_structureds_parse_json]: Done.

File: 1 [hunk 2]
```diff
@@ -0,0 +1,33 @@
+pick 4bce78b29c bugfix: make bitmap to vairiant more understandable
```
Comment (b41sh) [inline 1:1]: Looks like it was commited accidentally, please delete this file

Comment (ct20000901) [reply 1]: Done. Sorry for that.

**Review by b41sh — APPROVED** [review]

**Comment by b41sh** [comment]
LGTM, thanks for your contribution @ct20000901

**Comment by ct20000901** [comment]
> LGTM, thanks for your contribution @ct20000901

No problem. If there are any upcoming tasks or issues that need to be assigned, you can assign them to me as well.

**Comment by BohuTANG** [comment]
@ct20000901 

Thanks for the contribution 👍 

BTW, the email associated with your commit doesn't match your GitHub account email. As a result, the commit isn't being linked to your GitHub profile.
CleanShot 2023-10-10 at 11 49 02@2x

**Pull request merged**